deployment/cloud_run/test_swagger_debug.py

```
@@ -1,48 +1,53 @@
+
     print("\n=== Starting test server ===")
     print("Test these endpoints:")
     print("- http://localhost:5001/ (should work)")
     print("- http://localhost:5001/docs (should work)")
     print("- http://localhost:5001/api/health (should work)")
-    
-    app.run(host='0.0.0.0', port=int(os.environ.get('PORT', 5001)), debug=False)  # Debug mode disabled for security 
\ No newline at end of file
+
+    app.run(
+        host="0.0.0.0", port=int(os.environ.get("PORT", 5001)), debug=False
```

Ruff (0.13.1)

Possible binding to all interfaces

(S104)